### PR TITLE
YALB-495: Wire event paragraphs

### PR DESCRIPTION
**Pull request by ModulesUnraveled**
### Description of work
- Creates a handful of "default templates" to remove unwanted wrappers added by Stable9
- Wires the card and list view modes for events
- Wires the card and list views for events

### Functional testing steps:
- Follow the review steps here: https://github.com/yalesites-org/yalesites-project/pull/59

### CODE REVIEW

File: templates/node/node--event--card.html.twig [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+{% if content.field_event_format %}
+  {% set reference_card__overline -%}
+    {% include "@molecules/cards/reference-card/event/_event-format.twig" with {
+      format: content.field_event_format,
+    } %}
+  {%- endset %}
+{% endif %}
+
+{% embed "@molecules/cards/reference-card/reference-card.twig" with {
+  reference_card__heading: content.field_teaser_title,
```
Comment (nJim) [inline templates/node/node--event--card.html.twig:18]: Let's fall back on the node title if the teaser title is empty.

File: templates/node/node--event--card.html.twig [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  reference_card__subheading: date__formatted,
+  reference_card__url: url,
+  reference_card__snippet: content.field_teaser_text,
+  reference_card__image: 'true',
+} %}
+  {% block reference_card__image %}
+    {{ content.field_teaser_media }}
```
Comment (nJim) [inline templates/node/node--event--card.html.twig:25]: I'm going to make a follow up ticket to address the case when an image is not available. I think we will add a fallback image. No action needed at this time.

**Review by nJim — CHANGES_REQUESTED** [review]
Just one change requested. We are making the teaser content optional. The title field has the biggest impact on cards as the title is important for the design. So let's provide a fallback of the node title if the teaser title is not provided.

File: templates/node/node--event--list-item.html.twig [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+{% if content.field_event_format %}
+  {% set reference_card__overline %}
+    {% include "@molecules/cards/reference-card/event/_event-format.twig" with {
+      format: content.field_event_format,
+    } %}
+  {% endset %}
+{% endif %}
+
+{% embed "@molecules/cards/reference-card/reference-card.twig" with {
+  reference_card__heading: content.field_teaser_title,
```
Comment (nJim) [inline templates/node/node--event--list-item.html.twig:18]: Let's fall back on the node title if a teaser title is not provided.

File: templates/node/node--event--card.html.twig (hunk 1 above)
Comment (ModulesUnraveled) [reply templates/node/node--event--card.html.twig]: Good idea

File: templates/node/node--event--list-item.html.twig (hunk 3 above)
Comment (ModulesUnraveled) [reply templates/node/node--event--list-item.html.twig]: done

**Pull request merged**